### DISCUSSION TITLE
Error: No module named 'rosco'

### DISCUSSION THREAD

**Discussion by relhamoud** [NatLabRockies/ROSCO#373] [Q&A]
Hello everyone, 

I am a novice at python. I started by running the examples presented in: https://github.com/NREL/ROSCO/tree/main/Examples

I am using spyder which i launch from anaconda on windows OS. 

I followed the steps written here : 

https://rosco.readthedocs.io/en/latest/source/install.html#rosco-anaconda-download

where i type the commands in anaconda prompt. 

I am using python 3.11.7 so i changed the line from: 

conda create -y --name rosco-env python=3.10

to 

conda create -y --name rosco-env python=3.11.7

and everuthing was done successfully.

However when i try to run the 01_turbine_model.py, I get the following error in IPython console:

ModuleNotFoundError: No module named 'rosco'

I am confused because i did not understand this error and dont know how to solve it although i have already downloaded the package.

Could you help please ?

Thank you in advance.

Best Regards,

**Comment by relhamoud** [comment]
Thanks God, i resolved the problem !!!! Here are the commands:

conda config --add channels conda-forge 
conda create -y --name rosco-env python=3.10 (replace 3.10 by your python version)
conda activate rosco-env
conda install spyder

Then, to run the examples:
1- open anaconda navigator
2- choose installed applications and choose rosco-env as shown in the following figure


![InkedAnnotation 2024-08-15 121945_LI](https://github.com/user-attachments/assets/d9714f81-d353-41f9-9f7a-41b004d779fd)

3- launch Spyder

4- Execute 01_turbine_model.py

![InkedAnnotation 2024-08-15 122410_LI](https://github.com/user-attachments/assets/1ac69ef5-be58-482f-984f-ef9568706c0a)


5- Enjoy ! 😁

**Reply by christiannvaughn** [reply]
thanks for sharing
